cmd/kubeadm/app/master/manifests.go

```diff
@@ -133,6 +158,24 @@ func etcdVolumeMount() api.VolumeMount {
 	}
 }
 
+// certsVolume exposes host SSL certificates to pod containers.
+func certsVolume(s *kubeadmapi.KubeadmConfig) api.Volume {
+	return api.Volume{
+		Name: "certs",
+		VolumeSource: api.VolumeSource{
+			// TODO make path configurable
```

Comment:
I suppose we could just check the dirs in which given files are... let's discuss later on.